format/FormatNexusEigerDLS16M.py

```diff
@@ -5,31 +5,29 @@
 import libtbx
 
 from dxtbx.format.FormatNexusEigerDLS import FormatNexusEigerDLS
+from dxtbx.format.nexus import h5str
 from dxtbx.masking import GoniometerMaskerFactory
 from dxtbx.model import MultiAxisGoniometer
 
 VALID_NAMES = {
-    b"DIAMOND BEAMLINE I03",
-    b"DIAMOND BEAMLINE I04",
+    # "long" names
+    "DIAMOND BEAMLINE I03",
+    "DIAMOND BEAMLINE I04",
+    # "short" names
+    "DLS I03",
+    "DLS I04",
```

Comment:
Wouldn't this do the same thing but without the confusion of two separate lists that are always merged later anyway?
```suggestion
    "DLS I04",
    # "legacy" names
    "I03",
    "I04",
```

Reply:
I hold no strong opinion on this.

Reply:
Idem